metricflow/test/fixtures/sql_clients/adapter_backed_ddl_client.py

```diff
@@ -63,7 +64,12 @@ def create_table_from_dataframe(
                     elif type(cell) in [str, pd.Timestamp]:
                         # Wrap cell in quotes & escape existing single quotes
                         escaped_cell = self._quote_escape_value(str(cell))
-                        cells.append(f"'{escaped_cell}'")
+                        # Trino requires timestamp literals to be wrapped in a timestamp() function.
+                        # There is probably a better way to handle this.
```

Comment:
Given the nature of this function, I don't know of a better approach. This is for testing purposes only so as long as things work it should be fine.

Longer term we will move away from this method of loading test data, because it's brittle and unpleasant to work with, it just hasn't been a priority.